src/wazuh_modules/vulnerability_scanner/src/vulnerabilityScannerFacade.cpp

```diff
@@ -29,6 +31,81 @@ constexpr auto DELAYED_EVENTS_QUEUE_PATH = "queue/vd/delayed";
 constexpr auto SLEEP_RETRY_THREADS_SEC = 60;
 
 int SOCKET_WAIT = 0;
+constexpr auto COMPRESSED_DB_PATH {"vd_1.0.0_vd_4.8.0.tar.xz"};
```

Comment:
Where is used this variable ?

Reply:
Forgot to delete it after regex and filesystem iterator logic. Everything related to that, reverted.